docs_user/rhoso_attributes.adoc

```diff
@@ -5,11 +5,11 @@
 // Example: Use {osp_long} to display "OpenStack Platform".
 
 // RHEL version attributes
```

Comment:
@klgill I'm curious if we need this file at all? We have the `rhel_*_ver` variables in adoption-attributes.adoc too, and they were correct there but the settings got overriden by the incorrect values in this file. We should have the variables in one place only for each variant (upstream vs. downstream).